build-system/pr-check/e2e-tests.js

```diff
@@ -42,7 +42,7 @@ async function main() {
   if (!isTravisPullRequestBuild()) {
     downloadDistOutput(FILENAME);
     timedExecOrDie('gulp update-packages');
-    timedExecOrDie('gulp e2e --nobuild --headless');
+    timedExecOrDie('gulp e2e --nobuild --headless --compiled');
```

Comment:
This guy too please

https://github.com/ampproject/amphtml/blob/master/build-system/pr-check/experiment-tests.js

Reply:
I think I'm missing something :/. I see the `--compiled` flag in `experiment-tests.js`. Maybe we jinxed?

Reply:
Oh I missed it, nice!